cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -756,6 +765,30 @@ private String pgId(CQLTermNode node) throws QueryValidationException {
     }
   }
 
+  private String arrayNode(String index, CQLTermNode node, org.folio.rest.persist.ddlgen.Modifier relationModifier,
+    String modifierValue, DbIndex dbIndex, CqlModifiers modifiers) throws QueryValidationException {
+
+    StringBuilder res = new StringBuilder();
```

Comment:
The StringBuilder is not needed. Use String instead. The compiler will automatically use a StringBuilder in the background. Using String concatenation is much more easy to read than StringBuilder concatenation. https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java